src/test/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributesTest.java

```diff
@@ -47,6 +49,9 @@ public void testBuilder_buildNothing_shouldUseDefaultValues() {
         assertTrue(fsa.isOpeningEmailEnabled());
         assertTrue(fsa.isClosingEmailEnabled());
         assertTrue(fsa.isPublishedEmailEnabled());
+
+        assertEquals(new HashMap<>(), fsa.getStudentDeadlines());
+        assertEquals(new HashMap<>(), fsa.getInstructorDeadlines());
```

Comment:
Would it be possible to use the `isEmpty()` method for maps here?

Reply:
Hmm, I think not because it is necessary to show that it is using the non-null default value of a map, albeit an empty one. I am unsure if `assertTrue(fsa.getXXXDeadlines().isEmpty())` would show it is a map.

Reply:
Cool, that works good then